riposte-spi/src/test/java/com/nike/riposte/server/error/exception/Forbidden403ExceptionTest.java

```diff
@@ -0,0 +1,32 @@
+package com.nike.riposte.server.error.exception;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import java.util.UUID;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class Forbidden403ExceptionTest {
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@Test
+	public void should_honor_constructor_params() {
```

Comment:
There appear to be tabs mixed with spaces in this file (and maybe others?). Please fix so they're all spaces. Otherwise looks great!